yak/cmake/yak-config.cmake.in

```diff
@@ -0,0 +1,14 @@
+@PACKAGE_INIT@
+
+set(@PROJECT_NAME@_FOUND ON)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
+set_and_check(@PROJECT_NAME@_LIBRARY_DIRS "${PACKAGE_PREFIX_DIR}/lib")
+
+include(CMakeFindDependencyMacro)
+find_dependency(Eigen3)
+find_dependency(OpenMP)
```

Comment:
I think you also need to do the same check that the CMakeLists.txt does for the target and if it does not exist create it for upstream packages linking against yak.